ldi-core/ldes-client-v2/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/repository/sqlite/MemberRecordEntity.java

```diff
@@ -0,0 +1,47 @@
+package ldes.client.treenodesupplier.repository.sqlite;
+
+import ldes.client.treenodesupplier.domain.entities.MemberRecord;
+import ldes.client.treenodesupplier.domain.valueobject.MemberStatus;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFParserBuilder;
+
+import java.io.StringWriter;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+
+@Entity
+@NamedQuery(name = "Member.getByMemberStatus", query = "SELECT m FROM MemberRecordEntity m WHERE m.memberStatus = :memberStatus")
+@NamedQuery(name = "Member.countByMemberStatusAndId", query = "SELECT COUNT(m) FROM MemberRecordEntity m WHERE m.memberStatus = :memberStatus and m.id = :id")
+public class MemberRecordEntity {
+	@Id
+	private String id;
+	private MemberStatus memberStatus;
+	private String model;
```

Comment:
This a good example of the most confusing naming in my naming. 😕  This would be complete if there was a Jena Model named "string" 😄